Move scope parts to brevo config (#147)
vivid-planet/comet-brevo-module: .changeset/witty-coats-perform.md

---
"@comet/brevo-admin": major
---

> Define scopeParts in BrevoConfig
> 
> Previously the scopeParts had to be defined in the functions:
> 
> -   createBrevoContactsPage
> -   createTargetGroupsPage
> -   createEmailCampaignsPage
> 
> Now it has to be defined once in the BrevoConfig:

Reviewer: ```suggestion
Define `scopeParts` in `BrevoConfig`

Previously the `scopeParts` were passed to the functions:

-   createBrevoContactsPage
-   createTargetGroupsPage
-   createEmailCampaignsPage

Now they are defined once in the `BrevoConfig`:
```
